### PR TITLE
add showCloseButton prop

### PR DESCRIPTION
**Pull request by FullMoon04**
Attempt to implement #31

### CODE REVIEW

File: src/TourPortal.js [hunk 1]
```diff
@@ -290,13 +292,18 @@ class TourPortal extends Component {
...
       closeWithMask &&
-      !e.target.classList.contains(CN.mask.disableInteraction)
+      !e.target.classList.contains(CN.mask.disableInteraction) ||
+      !showCloseButton
     ) {
       onRequestClose(e)
     }
+    !showCloseButton && !closeWithMask &&
+      console.warn(
+        `You can't use 'closeWithMask' prop when 'showCloseButton' prop is false`
```
Comment (elrumordelaluz) [inline src/TourPortal.js]: I'm not sure about this `warn`. 

Could you please expan a little on why? Remember there is also the `esc` keyboard _trigger_ to close the _Tour_

**Comment by elrumordelaluz** [comment]
Could you please add the new `prop` on the [PropTypes table](https://github.com/elrumordelaluz/reactour#proptypes)?

File: src/TourPortal.js (hunk 1 above)
Comment (FullMoon04) [reply src/TourPortal.js]: As you think, I forgot `esc` keyboard trugger. Thank you.

Comment (elrumordelaluz) [reply src/TourPortal.js]: no prob!

**Comment by elrumordelaluz** [comment]
thanks @FullMoon04 for the PR, should be available in `reactour@1.10.0`

**Comment by FullMoon04** [comment]
@elrumordelaluz thanks!

**Pull request merged**